pkg/cmd/run/list/list.go

```diff
@@ -122,17 +135,18 @@ func listRun(opts *ListOptions) error {
 		tp.AddField(run.Name, nil, nil)
 		tp.AddField(run.HeadBranch, nil, cs.Bold)
 		tp.AddField(string(run.Event), nil, nil)
-
-		if opts.PlainOutput {
-			elapsed := run.UpdatedAt.Sub(run.CreatedAt)
-			if elapsed < 0 {
-				elapsed = 0
-			}
-			tp.AddField(elapsed.String(), nil, nil)
-		}
-
 		tp.AddField(fmt.Sprintf("%d", run.ID), nil, cs.Cyan)
 
+		elapsed := run.UpdatedAt.Sub(run.CreatedAt)
+		if elapsed < 0 {
+			elapsed = 0
+		}
+		tp.AddField(elapsed.String(), nil, nil)
+		age := time.Since(run.CreatedAt)
+		if age < 0 {
+			age = 0
+		}
+		tp.AddField(utils.FuzzyAgo(age), nil, nil)
```

Comment:
sorry, just one more request! this table has the potential to be very wide, so to save some space let's use `utils.FuzzyAgoAbbr` as in:

```suggestion
		tp.AddField(utils.FuzzyAgoAbbr(time.Now(), run.CreatedAt), nil, nil)
```